Formula/php.rb

```diff
@@ -0,0 +1,371 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net/"
+
+  url "https://php.net/get/php-7.2.1.tar.xz/from/this/mirror"
+  sha256 "6c6cf82fda6660ed963821eb0525214bb3547e8e29f447b9c15b2d8e6efd8822"
+
+  depends_on "pkg-config" => :build
+  depends_on "argon2"
+  depends_on "aspell"
+  depends_on "curl" if MacOS.version < :lion
+  depends_on "freetds"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gmp"
+  depends_on "httpd"
+  depends_on "icu4c"
+  depends_on "imap-uw"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libpq"
+  depends_on "libsodium"
+  depends_on "libzip"
+  depends_on "net-snmp"
+  depends_on "openssl"
+  depends_on "pcre"
+  depends_on "unixodbc"
+  depends_on "webp"
+
+  resource "enchant" do
```

Comment:
I think we should leave out enchant support until it can use `depends_on "enchant"`

Reply:
Then lets make it so. The has 815 unique events since Dec 1st 2017 so compared to something like intl which has 40k+ its use of course less and gmp which has around 4k unique events in the same time period.